src/Aspire.Dashboard/Configuration/DashboardOptions.cs

```diff
@@ -163,3 +164,52 @@ public sealed class TelemetryLimitOptions
     public int MaxAttributeLength { get; set; } = int.MaxValue;
     public int MaxSpanEventCount { get; set; } = int.MaxValue;
 }
+
+public sealed class OpenIdConnectOptions
```

Comment:
For consistency with other config classes in this file:

```suggestion
// Don't set values after validating/parsing options.
public sealed class OpenIdConnectOptions
```